attributes/default.rb

```diff
@@ -44,3 +43,9 @@
 default['flink']['parallelization']['degree']        = node['cpu']['total']
 default['flink']['webclient_port']                   = 8888
 default['flink']['taskmanager']['network_num_buffers']  = 2048
+
+#
+# Featurestore dependencies
+#
+default['flink']['hsfs']['version']                  = node['install']['version']
+default['flink']['hsfs']['url']                      = "#{node['download_url']}/hsfs/#{node['flink']['hsfs']['version']}/hsfs-flink-#{node['flink']['hsfs']['version']}.jar"
```

Comment:
I have a deja-vu here. I thought we discussed this when we merged hsfs-flink to not include it in the distribution as in any case users need to have the dependency on their maven build.

Also I don't see this attribute being used anywhere.

Reply:
ahh sorry, leftovers